roles/pulp-routes/tasks/main.yml

```diff
@@ -1,8 +1,30 @@
 ---
+- name: Load Route TLS certificate
+  include_tasks: load_route_tls_secret.yml
+  when:
+    - ingress_type | lower == 'route'
+    - route_tls_secret != ''
+
+- set_fact:
+    _pulp_plugins:
+      - { name: "{{ ansible_operator_meta.name }}-content", path: "/pulp/content/", targetPort: "content-24816", serviceName: "{{ ansible_operator_meta.name }}-content-svc" }
+      - { name: "{{ ansible_operator_meta.name }}-container", path: "/pulp/container/", targetPort: "content-24816", serviceName: "{{ ansible_operator_meta.name }}-content-svc" }
+      - { name: "{{ ansible_operator_meta.name }}-cookbook", path: "/pulp_cookbook/content/", targetPort: "content-24816", serviceName: "{{ ansible_operator_meta.name }}-content-svc" }
+      - { name: "{{ ansible_operator_meta.name }}", path: "/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
+      - { name: "{{ ansible_operator_meta.name }}-api-v3", path: "/pulp/api/v3/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
+      - { name: "{{ ansible_operator_meta.name }}-auth", path: "/auth/login/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
+      - { name: "{{ ansible_operator_meta.name }}-v2", path: "/v2/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
```

Comment:
can you make the name more explicit? for example `name: "{{ ansible_operator_meta.name }}-registry-v2"` , `registry-extentions-v2`, `registry-token`.
Ideally I would user `container-registry-` prefix but if it gets too long it might be not much readable.

Reply:
lgtm